packages/lit-html/src/lit-html.ts

```diff
@@ -1485,17 +1483,17 @@ export const _Σ = {
   _isIterable: isIterable,
   _resolveDirective: resolveDirective,
   // Used in tests and private-ssr-support
-  _ChildPart: ChildPartImpl,
-  _AttributePart: AttributePartImpl as AttributePartConstructor,
-  _BooleanAttributePart: BooleanAttributePartImpl as AttributePartConstructor,
-  _EventPart: EventPartImpl as AttributePartConstructor,
-  _PropertyPart: PropertyPartImpl as AttributePartConstructor,
-  _ElementPart: ElementPartImpl as ElementPartConstructor,
+  _ChildPart: ChildPart,
+  _AttributePart: AttributePart as AttributePartConstructor,
```

Comment:
Can you try removing these casts? Hopefully exporting the class types makes `typeof C` be a visible type to users of this object.

Reply:
Ok, I removed these, left another comment about something related below.